lib/authorizer/digest.js

```diff
@@ -1,5 +1,5 @@
 var _ = require('lodash'),
-    crypto = require('crypto-js'),
```

Comment:
Do we know why it was -js earlier and what is the implication of changing this?

Reply:
`crypto-js` was used earlier because Node's `Crypto` is not supported in browser(for chrome app). But now we are anyway using Node's crypto in other auths for algorithms like `RSA` which is not there in `crypto-js`.

And if we need to support it in browser, Browserify takes care of it using [crypto-browserify](https://github.com/crypto-browserify/crypto-browserify). So now `crypto-js` seems redundant dependency.